### DISCUSSION TITLE
デザイン　ヒアリング

### DISCUSSION THREAD

**Discussion by txxxxc** [lit-kansai/dj_system-docs#58] [ドキュメント]
ヒアリング頼んだ！！！

**Comment by txxxxc** [comment]
### DJ SystemのDJは誰なのか・誰だと思っているのか

**Reply by txxxxc** [reply]
- MCがDJではある
- メンバーはDJに曲をリクエストするみたいな感じ
- DJは曲を選ぶことも仕事なので、曲を選んでいるメンバーはDJになりうる説はあるけど、あくまでDJにメンバーがリクエストするみたいな流れを想定してます

**Reply by mizucoffee** [reply]
DJの主体はMC、マインドとしてはりょうさんのDJをイメージしている。

**Reply by 9-sho-5** [reply]
MC向けにお便り送ったりしてるのでDJ主体はMCのイメージ

**Reply by gassi4731** [reply]
概念。
ネームバリューとしての使い方。

MC＝DJ説とすると、
MCが、曲選択をしなくなる代行サービスになるので、MCではなさそう。

```
ディスクジョッキーまたはDJとは、ポピュラー音楽をラジオ番組などで選曲、操作する、または演奏会場やクラブ、ソフトなどで再生機器で客に聴かせる人物。
```
らしいので、DJが誰？となるとリクエストしているメンバー自身になりそう。

**Comment by txxxxc** [comment]
### キャンプとかスクールの中でのDJ Systemはどういう位置づけなのか、位置づけにしたいのか

(曲はすでに流れている中で、DJ Systemを導入することによってどういう価値があるのか)

**Reply by txxxxc** [reply]
- キャンプ・スクールでプログラミングを学ぶことで行えるアウトプットの例という位置づけ
- ものづくりを通して身の回りの世界を変えている具体例として捉えてほしい
- 身近なメンターが作った作品という位置づけ(これは関西限定かも)
- メンバーからキャンプ・スクールの会場で自己表現が出来る機会

**Reply by mizucoffee** [reply]
- 班と会場を繋げる、数ある手法の中の内の"1つのコンテンツ"として解釈している@キャンプ
  - 自分のアクションが会場に影響するという体験
- 使うか使わないかはMCに依存して良い
- スクールに関しては特に意味なく、好きな音楽流せるコンテンツ位の軽い気持ちで良いと思う

**Reply by 9-sho-5** [reply]
- 会場内のメンバーの出来事(お便り)をDJが話すコミュニケーションツールの１つ
- 「作る」から「使う」道を見せる一つ

**Reply by gassi4731** [reply]
**会場に開発メンバーがいない場合**
個人と会場を繋げるためのツール。
メンターが作ったんだー！で少しでも身近に感じてもらいやすい。メンター=すごい！みたいな気持ちを作りたい。

**会場に開発メンバーがある場合**
個人と会場を繋げるためのツール。
身近な存在(会場内にいる！)人が作ったんだ！という体験。
技術的要素やデザイン要素を喋れるので、1つの追加コンテンツになるかも

**Comment by txxxxc** [comment]
### ターゲットに何を届けたいのか(技術視点)

**Reply by txxxxc** [reply]
- (できるだけ)キャンプやスクールで使われている技術を使って作りたい
- シンプルなUIでスムーズにユーザーに使ってもらいたい

**Reply by mizucoffee** [reply]
対メンバー
- ストレス無く使えるはマスト
- 今更だけどキャンプ技術である必要はさして無い気はする
  - 身近に感じてもらえるけどそれを実現するためのコストがやや重い
  - DJ MIYUKKI、Firebaseやし（小声）
    - なんならそれでFirebase知った民なので笑

対MC
- どう考えてもリアルタイム更新はほしい

**Reply by 9-sho-5** [reply]
- 学んでいる技術の１ゴール
- 作ってきたロードマップ的なものとか(１つの開発手法として見せる)

**Reply by gassi4731** [reply]
セキリュティが確保されて、ストレスなく使えればGood。

技術は明らかに高度になっているので、技術的制約はこの観点では無くして良さそう。

**Comment by txxxxc** [comment]
### ターゲットに何を届けたいのか(体験視点)

どういう気持ちになってほしいのか

**Reply by txxxxc** [reply]
- インタラクティブなメディアアートなど、普段身の回りで使うサービスには無い部分で差別化して特別感を演出したい
- 自分がリクエストした曲・好きな曲が流れることによって会場に参加している感をもっと感じてほしい
- 室の高いサービスを触ることによってプログラミングに対するモチベーションにしてほしい

**Reply by mizucoffee** [reply]
↑概ね同意

対メンバー
- 「このサービスかっけえ！・すげぇ！」って思ってほしい所はある

対MC
- このサービスを使った結果を会場のコンテンツとして昇華しやすい機能がほしい

**Reply by 9-sho-5** [reply]
- え！すげぇのインパクト
- リクエスト(曲、お便り)が流れる、読まれるワクワク感
- メンバーの色々を知るきっかけ
- 作るものから使う人を考える

**Reply by gassi4731** [reply]
リクエストした曲が実際に流れるという体験が大事。

↑他は、同意見
